kilkari/src/main/java/org/motechproject/nms/kilkari/domain/RejectionReasons.java

```diff
@@ -6,7 +6,6 @@
 public enum RejectionReasons {
     FLW_TYPE_NOT_ASHA,
     INVALID_LOCATION,
-    MOBILE_NUMBER_ALREADY_IN_USE,
```

Comment:
why do I just see removal o fMOBILE_NUMBER_ALREADY_IN_USE from the enum, but not the other reason added to this ?

Reply:
looks like new one is already present !!